tests/run/pure_fused.py

```diff
@@ -37,6 +40,7 @@ def cpfunc(self, arg):
         >>> TestCls().cpfunc(2)
         'int'
         """
+        loca = arg
```

Comment:
Extraneous "a"?

Reply:
Yes. I think I wanted a different name to catch in the debugger when preparing this. It'll still work like this but I'll fix it if other changes are needed.

Reply:
Ok cool.